yotta/version.txt

```diff
@@ -1 +1 @@
-0.16.4
```

Comment:
The intention was to make a new release right after this change is approved, but sure, this can be part of another PR if you think that's better.